.travis.yml

```diff
@@ -1,25 +1,37 @@
-language: c
+language: cpp
```

Comment:
cmake will require a c++ compiler to be detected, even if it doesn't use it (it is only used if WITH_ASTYLE is ON, which is not the default). There's perhaps a way to restrict c++ compiler detection only when WITH_ASTYLE is ON but I couldn't find it.
The C compiler (gcc) will still be used to compile the .c files.

Anyway we require CMake to be able to build openjpeg and CMake source code itself is C++, so the platform must have a working C++ compiler.

And at some point we might perhaps want to use C++ for openjpeg itself.